app/controllers/api/v3/infrastructures_controller.rb

```diff
@@ -138,16 +138,25 @@ def authorize_by_username
   def profile_by_app_group_name
     @app_group = AppGroup.find_by(
       name: params[:app_group_name],
+      status: "ACTIVE"
     )
 
-    if @app_group.blank? || !@app_group.ACTIVE?
+    if @app_group.blank? || @app_group.status != "ACTIVE"
       render(json: {
                success: false,
                errors: ['App Group not found'],
                code: 404,
              }, status: :not_found) && return
     end
 
+    if @helm_infrastructure.blank?
```

Comment:
I don't think we need this right? where @helm_infrastructure defined?

Reply:
in my case when testing in local, its happen if there are appgroups that have provisioning status `PENDING` and status `INACTIVE`, . if i remove this, it will throw a random html response if i hit the appgroup that have condition like that.

Reply:
what's this random html? is it 404 or 500? if it is 404, I guess it is expected because it is still not active, wdyt @fadlinurhasan

Reply:
its 404, make sense, will update it